core/templates/dev/head/pages/state_editor/state_content_editor_directive.html

```diff
@@ -1,3 +1,13 @@
+<div class="shadow-preview-card" aria-hidden="true">
+  <div class="oppia-learner-view-card-content">
+    <div class="oppia-learner-view-card-top-section">
+      <angular-html-bind class="rte-viewer oppia-learner-view-card-top-content"
```

Comment:
I don't understand the purpose of this code, can you please explain?

Reply:
Hi @DubeySandeep, this code creates an off-screen preview of the learner view card from the content user is typing in the editor mode to find the height of learner view card. This is similar to what is done in learner view to find the height of next card.

I have added a comment now to make this clear, PTAL!

Reply:
Ah, I see. Thanks for the explanation.